configs/myconfigs/sample.py

```diff
@@ -52,6 +52,13 @@
 
 SimpleOpts.add_option("--script", default='',
                       help="Script to execute in the simulated system")
+SimpleOpts.add_option("--warmup_time", default="10ms",
+                      help="Simulated functional warmup time in ms")
```

Comment:
No need for "in ms". In fact, there is no requirement on "ms". It could be "us", or just "s".